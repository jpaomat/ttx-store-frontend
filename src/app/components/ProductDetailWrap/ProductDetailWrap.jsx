import './ProductDetailWrap.scss';

export const ProductDetailWrap = () => {
    return (
        <>
            <h2>Hola detaless card</h2>
        </>
    );
};
