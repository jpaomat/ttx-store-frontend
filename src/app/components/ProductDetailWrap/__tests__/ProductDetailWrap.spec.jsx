import Renderer from 'react-test-renderer/shallow';
import { ProductDetailWrap } from '../ProductDetailWrap';

const renderer = new Renderer();

describe('Product detail wrap component', () => { 
    it('Renderer componen', ()=> {
        const component = renderer.render(<ProductDetailWrap />);
        expect(component).toMatchSnapshot();
    });
 });