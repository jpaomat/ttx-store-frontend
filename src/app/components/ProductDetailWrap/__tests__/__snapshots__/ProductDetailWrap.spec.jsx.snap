// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product detail wrap component Renderer componen 1`] = `
<React.Fragment>
  <h2>
    Hola detaless card
  </h2>
</React.Fragment>
`;
